OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -811,6 +869,590 @@ namespace OloEngine
+                light.Ambient = dirLight.m_Color * 0.1f;
+                light.Diffuse = dirLight.m_Color * dirLight.m_Intensity;
+                light.Specular = dirLight.m_Color * dirLight.m_Intensity;
+                Renderer3D::SetLight(light);
+                Renderer3D::SetViewPosition(camera.GetPosition());
+                break; // Only use first directional light for now
+            }
+        }
+
+        // Draw mesh entities (skip animated entities - they're rendered separately)
+        {
+            auto view = m_Registry.view<TransformComponent, MeshComponent>();
```

Clang (14.0.6)

[warning] multiple declarations in a single statement reduces readability

(readability-isolate-declaration)